src/Renderer/ThreeExtended/GeoJSON2Three.js

```diff
@@ -291,13 +332,15 @@ function readFeatureCollection(crsIn, crsOut, json, filteringExtent) {
 }
 
 export default {
-    parse(crsOut, json, filteringExtent) {
-        const crsIn = readCRS(json);
+    parse(crsOut, json, filteringExtent, options = { toMesh: true }) {
```

Comment:
I need to think more about this but I don't like this change because using a module named `GeoJSON2Three` to *not* produce a three object is kind of weird.

You probably want to split this file in 3 files: a Geojson parser producing *something*, a `somethingToThree` module and a `somethingToTexture` module.

Reply:
I agree with you
We need to valid architecture before I start the changes

Reply:
I'd start with the suggestion I made earlier and see if it works (using the object defined at https://github.com/iTowns/itowns/pull/450/files#diff-6e5ed93be78417d38f4b8eb6d6f2a5e1R52 as the intermediate representation)

Reply:
@peppsac : Ok, done, I split GeoJSON2Three, the Pr is ready